test/install-jasmine-gjs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -ex
+
+cd /tmp
```

Comment:
Please use `mktemp -D` instead of a hardcoded path where we might run into conflicts.

Reply:
Good idea, will do

Reply:
Done